src/main/java/at/hannibal2/skyhanni/features/garden/GardenPlotAPI.kt

```diff
@@ -155,6 +159,14 @@ object GardenPlotAPI {
 
             plot?.setSpray(spray, 30.minutes)
         }
+
+        portableWasherPattern.matchMatcher(event.message) {
+            for (plot in plots) {
+                if (plot.currentSpray != null) {
```

Comment:
no need for this extra check. just reset everything

Reply:
I added this because otherwise it notified that the spray was cleared on all plots from 1 to 24. But we're going to be removing that anyway in favor of Hypixel's own notification, so I guess it can be removed.

Reply:
Oh, I have overseen this extra behavior.

Reply:
I've kept this for now because it seems like we have not removed the notification yet.